docs/advanced/accesstransformers.mdx

```diff
@@ -10,16 +13,67 @@ Adding ATs
 
 Adding an Access Transformer to your mod project is as simple as adding a single line into your `build.gradle`:
 
+<Tabs defaultValue="latest">
+<TabItem value="latest" label="Latest">
+Access Transformers need to be declared in both `build.gradle` and `mods.toml`:
+
+```groovy
+// In build.gradle:
+// This block is where your mappings version is also specified
+minecraft {
+  accessTransformers {
+    file('src/main/resources/META-INF/accesstransformer.cfg')
+  }
+}
+```
+
+```toml
+# In mods.toml:
+[[accessTransformers]]
+file="META-INF/accesstransformer.at"
+```
+
+AT files can be anywhere specified by the lines above, though NeoForge will default to searching for `META-INF/accesstransformer.cfg` if no other file is specified.
+
+Additionally, multiple AT files can be specified and will be applied in order. This can be useful for larger mods with multiple packages.
+
+```groovy
+// In build.gradle:
+minecraft {
+  accessTransformers {
+    file('src/main/resources/accesstransformer_main.cfg')
+    file('src/additions/resources/accesstransformer_additions.cfg')
+  }
+}
+```
+
+```toml
+# In mods.toml
+[[accessTransformers]]
+file="accesstransformer_main.at"
+
+[[accessTransformers]]
+file="accesstransformer_additions.at"
```

Comment:
I think you meant .cfg not .at (also in the other file)

Reply:
I'll correct that as well, thank you. I was basing it off of the 20.3 patch notes, which also used `.at` instead of `.cfg`, but I see that the Gradle and `mods.toml` specifications should match.